resources/models/reservation.php

```diff
@@ -221,25 +221,52 @@
             'mode' => 'time',
             'span' => 'right',
         ],
+        'registered_customer_id' => [
+            'label' => 'lang:igniter.reservation::default.text_customer',
+            'type' => 'select',
+            'options' => [Igniter\User\Models\Customer::class, 'getDropdownOptions'],
+            'span' => 'left',
```

Comment:
You can remove this so that field has 100% width to avoid unnecessary layout shifts